scripts/Deepcell_Preprocessing_MIBItiff.ipynb

```diff
@@ -87,7 +84,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.6.10"
```

Comment:
Out of curiosity, are we planning on sticking with Python 3.6 as outlined in the Dockerfile for the near future?

Reply:
Yes, I think we'll stick with 3.6 until we find a package that requires 3.7. But we can add 3.7 to Travis